site.yml

```diff
@@ -1,4 +1,48 @@
 ---
-- include: redhat_release.yml
-- include: autotested.yml
-- include: cleanup.yml
+
+# Setup/override global variables
+- hosts: all
+  tasks:
+    # Defines adept_job_no_group and rt_adept_job_no
+    - include: roles/common/tasks/adept_job_no.yml
+
```

Comment:
I considered that, however it has a higher precedence (http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable) and I need to be able to override the place-holder value (by a ``set_facts``).  It's kind of a subtle "gotcha" that's bitten me before, so I prefer to always go for lowest-precedence possible.  In fact, there are some items in ``files/variables.yml`` that I should move to ``group_vars/all.yml``.

Also, ``site.yml`` is kind of just an "example", I'd like to keep most things modular so they can be plugged in different ways in alternate play-books depending on the situation/environment.